### PR TITLE
Fixed makefile and brainwash.cpp

### PR DESCRIPTION
**Pull request by dacianf**

### CODE REVIEW

File: BrainWash.cpp [hunk 1]
```diff
@@ -113,7 +114,7 @@ class ScreenResMenuField : public BMenuField
 public:
 	ScreenResMenuField(BRect frame, const char *label, uint32 res,
 					   uint32 *target_value = 0)
-		: BMenuField(frame, 0, label, new BPopUpMenu("resolution"))
+		: BMenuField(frame, NULL, label, (BMenu*)(new BPopUpMenu("resolution")))
```
Comment (mmuman) [inline BrainWash.cpp]: The BMenu * cast seems useless, BPopUpMenu is a BMenu subclass.

**Comment by mmuman** [comment]
For the commit log, one usually use the imperative "Fix ..." or the past "Fixed ...", but not the at the first person. (We know it's you who did it, it's in the author field ;-)

File: BrainWash.cpp [hunk 2]
```diff
@@ -22,6 +22,7 @@
 #include "Functions.h"
 #include "Dimensions.h"
 #include "Threads.h"
+#include <PopUpMenu.h>
```
Comment (humdingerb) [inline BrainWash.cpp:8]: You can remove all the app/, interface/, game/ above and sort PopUpMenu.h into it alphabetically. In fact sort all includes alphabetically while you're at it.

File: makefile [hunk 3]
```diff
@@ -94,6 +94,7 @@ COMPILER_FLAGS =
 LINKER_FLAGS =
 
 
-## include the makefile-engine
-include /boot/develop/etc/makefile-engine
-
+## Include the Makefile-Engine
+DEVEL_DIRECTORY := \
+	$(shell findpaths -r "makefile_engine" B_FIND_PATH_DEVELOP_DIRECTORY)
+include $(DEVEL_DIRECTORY)/etc/makefile-engine
```
Comment (humdingerb) [inline makefile:100]: line 17: screen savers are actuall of type "SHARED".
line 83: remove the TRUE

**Comment by scottmc** [comment]
While you are at it, please also use "rc -d BrainWash.rsrc" to create a BrainWash.rdef and then add the .rdef to the makefile, in place of the .rsrc file.  See https://www.haiku-os.org/documents/dev/compile_them_resources

**Review by humdingerb — CHANGES_REQUESTED** [review]

**Comment by scottmc** [comment]
Also remove the "_APP_" symlink from the repo, as this won't be needed, as the install step will be handled by the recipe.

File: makefile (hunk 3 above)
Comment (dacianf) [reply makefile:100]: @humdingerb what is about those line from 97-100?

Comment (humdingerb) [reply makefile:100]: Github lets me only comment on lines that were changed in the PR. Being lazy, I just clicked on a changed line and specified the lines I had comments on in the comment...

**Pull request merged**